packages/web-components/fast-components/src/dialog/dialog.definition.ts

```diff
@@ -11,42 +11,46 @@ export const fastDialogDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "modal",
+                    title: "Modal",
                     type: DataType.boolean,
-                    description: "The model attribute",
+                    description:
+                        "When modal, user interaction will be limited to the contents of the element",
                     default: true,
                     required: false,
                 },
                 {
                     name: "hidden",
-                    description: "The hidden attribute",
+                    title: "Hidden",
+                    description: "The hidden state of the element",
                     type: DataType.boolean,
                     default: false,
                     required: false,
                 },
                 {
                     name: "trap-focus",
-                    description: "The trap-focus attribute",
+                    title: "Trap focus",
+                    description: "Indicates that the dialog should trap focus",
                     type: DataType.boolean,
                     default: true,
                     required: false,
                 },
                 {
                     name: "aria-describedby",
-                    description: "The aria-describedby attribute",
+                    description: "The HTML aria-describedby attribute",
                     type: DataType.string,
                     default: false,
                     required: false,
                 },
                 {
                     name: "aria-labelledby",
-                    description: "The aria-labelledby attribute",
+                    description: "The HTML aria-labelledby attribute",
                     type: DataType.string,
                     default: false,
                     required: false,
                 },
                 {
                     name: "aria-label",
-                    description: "The aria-label attribute",
+                    description: "The HTML aria-label attribute",
```

Comment:
```suggestion
                    title: "Accessibility label",
                    description: "The HTML aria-label attribute",
```

Reply:
I actually think designers are fairly familiar with aria-label - accessibility label is broad and technically the label for dialog could be any of the three.